src/amo/components/MastHead.scss

```diff
@@ -0,0 +1,66 @@
+@import "~amo/css/inc/vars";
+
+#tabzilla {
+  left: 14px;
+  position: absolute;
+}
+
+.MastHead {
+  background: $header-background;
+  min-height: 100px;
+  position: relative;
+}
+
+.MastHead-header {
+  padding-top: 80px;
+  padding-bottom: 30px;
+  text-align: center;
+}
+
+.MastHead-title {
+  align-items: center;
+  color: $header-font-color;
+  display: flex;
+  font-size: 24px;
+  justify-content: center;
+
+  &:link {
+    font-size: 24px;
+    text-decoration: none;
+  }
+
+  &::before {
+    background: url(../img/icons/fox-puzzle.svg) no-repeat 50% 50%;
```

Comment:
Here your URL has no quotes, but below it has quotes. (https://github.com/mozilla/addons-frontend/pull/1127/files#diff-6ce0883413606969405448711b59e8baR61)

I think I prefer quotes, but ultimately just want consistency 😄